src/agent.rs

```diff
@@ -595,7 +653,7 @@ impl TxBuffer {
         let (msg1, msg2) = self.mbufs.as_mut_slices();
         let mut sent = 0_u16;
         let mut unsent = true;
-
+        // XXX pop_front???
```

Comment:
What's meaning of this comment?

Reply:
Because later I used `pop_front` to remove sent packets. I was afraid that the order of `VecDeque::pop_front` was different from `VecDeque::as_mut_slices`. Then the problem was eliminated and I forgot to delete the comment...